common/SmokeTests/SmokeTest/SmokeTest.csproj

```diff
@@ -6,11 +6,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.0.0-preview.4" />
-    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.0.0-preview.3" />
-    <PackageReference Include="Azure.Security.Keyvault.Secrets" Version="4.0.0-preview.4" />
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.0.0-preview.3" />
-    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.6.0" />
+    <PackageReference Include="Azure.Identity" Version="1.0.0-preview.5" />
+    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.0.0-preview.4" />
+    <PackageReference Include="Azure.Security.Keyvault.Secrets" Version="4.0.0-preview.5" />
```

Comment:
We GA'd. Why use the preview.5 version?

Reply:
This PR was left behind a while back. I wanted to make sure it was updated to show that the October preview also passed smoke testing. The GA smoke tests is coming in https://github.com/Azure/azure-sdk-for-net/issues/8630